lib/routes/key_data.js

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Joi = require('joi');
+
+const AppError = require('../error');
+const db = require('../db');
+const logger = require('../logging')('routes.key_data');
+const P = require('../promise');
+const validators = require('../validators');
+const verify = require('../browserid');
+const Scope = require('../scope');
+
+const ADDITIONAL_KEY_MATERIAL = Buffer.alloc(32).toString('hex');
```

Comment:
It may be worth a comment here to say that we're using a static value on purpose, and in future we can read it from the DB.  Otherwise a casual reader (e.g. the various people inside Mozilla who've been keeping one eye on this proposal) might worry that we're doing something wrong here.